src/data/news/news.yml

```diff
@@ -193,6 +493,14 @@
     sortDate: "2020-09-12"
     icon: "dcrInMedia.svg"
     externalSource: "medium.com/decred/journals/home"
+-
+  Title: "Decred Release v1.5.2"
+  Author: "David Hill"
```

Comment:
I wonder if jrick was building those earlier releases too?

Reply:
idk, I just went with the name of whoever published the release on github, which doesnt necessarily correspond with who did the build.